install/kubernetes/helm/istio/charts/sidecar-injector/templates/mutatingwebhook.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Capabilities.APIVersions.Has "admissionregistration.k8s.io/v1beta1" -}}
+{{- if .Values.global.sidecarInjectEnabled }}
```

Comment:
please don't remove the api check. `global,sidecarInjectEnabled` should no be checked here, since this check should be on `requirements.yaml`

Reply:
ah thanks, I missed that!

Reply:
only global variables right now are `securityEnabled` and `rbacEnabled` because they are needed on different charts. For the rest, the `enabled` flag doesn't have to be global.

Reply:
ok I'm proposing to have this as false by default and delete the other var.  agreed?